doc/install_ubuntu.txt

```diff
@@ -14,6 +14,7 @@ Ubuntu Installation Instructions
 .. _gpu_linux:
 
 .. |PlatformCompiler| replace:: ``python-dev``, ``g++`` >= 4.2
+.. |CompilerName| replace:: ``g++``
```

Comment:
I use `|CompilerName|` as a replacement string into `requirements.inc`, to indicate compiler name for a specific platform (`g++` for Windows and Linux, `Clang` for Mac).